layer/tests/tests_promoted.cpp

```diff
@@ -33,17 +33,9 @@ static const char* CONFIG_PATH = "bin/Release";
 static const char* CONFIG_PATH = "lib";
 #endif
 
-struct TestInit {
-    TestInit() : instance(VK_NULL_HANDLE), physical_device(VK_NULL_HANDLE) {}
-
-    ~TestInit() {
-        if (this->instance != VK_NULL_HANDLE) {
-            vkDestroyInstance(this->instance, nullptr);
-            this->instance = VK_NULL_HANDLE;
-        }
-    }
-
-    void init() {
+class layer_promoted : public VkTestFramework {
```

Comment:
Should it be `LayerPromoted`?

Reply:
Yes!  All function names and class names should be UpperCamelCase.  Sure, I'll be glad to change these!